modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -72,9 +73,10 @@ def map_downstream_error(status, body)
     end
 
     def download_raw_file(guid)
```

Comment:
Might be worth a comment indicating that this returns a tempfile that should be closed. Would like to move toward providing documentation for vets-api internals via rdoc or something similar eventually.

Reply:
Probably worth confirming the caller currently closes the tempfile.

Reply:
Was not being explicit about it, fixed